src/lib/gundb/UserStorageClass.js

```diff
@@ -1237,7 +1237,7 @@ export class UserStorage {
       .get(field)
       .get('value')
       .decrypt()
-      .catch(e => logger.error('getProfileFieldValue decrypt failed:', e.msg, e))
+      .catch(e => logger.error('getProfileFieldValue decrypt failed:', e.message || e.msg, e))
```

Comment:
.decrypt could throw an Error object or just a string. in the case of string let's create an Error object

```suggestion
      .catch(reason => {
        let exception = reason
        let { message } = exception
        
        if (!isError(reason)) {
          message = reason
          exception = new Error(reason)
        }
        
        logger.error('getProfileFieldValue decrypt failed:', message, exception)
      })
```